### DISCUSSION TITLE
What is the difference between the wotlk_classic branch and the 3.3.5 branch?

### DISCUSSION THREAD

**Discussion by binzlink** [TrinityCore/TrinityCore#28764] [Q&A]
Hi,All

I use wotlk_classic branch to build server.

I enter the database and get the gamebuild value is 47168.

I have a client which version is 3.4.0(47168).

But i stuck at login page.

Does anyone have solutions?

![截屏2023-01-18 19 36 20](https://user-images.githubusercontent.com/1628140/213161604-195cf30f-5d67-4b88-a719-d705ce2fbdbd.png)

PS: 
Macbook Air 
MacOS: 13.0 (22A380)
Intel Core i5

**Comment by Jinnaix** [comment]
wotlk_classic is based on master branch, so you want to follow the final steps for master in our wiki.
https://trinitycore.info/en/install/Client-Setup

**Reply by binzlink** [reply]
> wotlk_classic is based on master branch, so you want to follow the final steps for master in our wiki. https://trinitycore.info/en/install/Client-Setup

yes.I have set the portal value to 127.0.0.1.It's not working

**Comment by Ovahlord** [comment]
you also need a client launcher like the one Arctium provides

**Comment by memebaba** [comment]
How do we extract the maps? When I used the wotlk_classic branch extractors against the retail/classic client, it has a bunch of missing files.

**Reply by mdX7** [reply]
> How do we extract the maps? When I used the wotlk_classic branch extractors against the retail/classic client, it has a bunch of missing files.

Current classic @ blizz is 3.4.1, wotlk_classic is currently at 3.4.0, not updated yet
